source/Engine.cpp

```diff
@@ -1621,12 +1624,19 @@ void Engine::HandleKeyboardInputs()
 	// If holding JUMP or toggling LAND, also send WAIT. This prevents the jump from
 	// starting (e.g. while escorts are aligning), or switches the landing target.
 	if(keyHeld.Has(Command::JUMP) || (keyHeld.Has(Command::LAND) && landKeyInterval < landCooldown))
-		activeCommands.Set(Command::WAIT);
-	else
-		activeCommands.Clear(Command::WAIT);
+		activeCommands |= Command::WAIT;
 	
-	// Transfer all newly pressed, unhandled keys to active commands.
+	// Transfer all newly pressed unhandled keys to active commands, except for BACK, since
```

Comment:
What's wrong with the original comment here?

(Also, `autoPilot` in the comment below should really be autopilot and not the name of a variable.)

Reply:
I think that was my doing, from the pull request that petervdmeer merged into his branch.  I have no strong opinions about the exact wording and capitalization of a comment.

Reply:
Doesn't matter a lot who did what. I fixed both comments in the latest commit.